todos/push-fcm-proposal.md

```
@@ -0,0 +1,589 @@
+- `voice_actor_subscriptions` schema
+- `manage-subscription.post.ts`
+- `handleDeepLink` util
+- All voice-actor UI / settings / profile views
+- All auth middleware
+
+---
+
+## 9. Configuration surface
+
+`.env.example`:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)